packages/server/Dockerfile

```diff
@@ -1,18 +1,23 @@
-FROM node:16.15-bullseye-slim as build-stage
 ARG NODE_ENV=production
 ARG SPECKLE_SERVER_VERSION=custom
+ARG FILE_SIZE_LIMIT_MB=100
+ARG NODE_ENV=production
 
-RUN apt-get update && \
-  DEBIAN_FRONTEND=noninteractive apt-get install -y \
-  --no-install-recommends \
-  tini=0.19.0-1 \
-  fonts-dejavu-core=2.37-2 \
```

Comment:
fonts-dejavu-core & fontconfig have been removed by this PR as this allows us to remove apt as a dependency (tini is installed manually by wget & un-tarring)
@gjedlicska @fabis94 - do you know why they were installed?  If they're required, I can look to manual installation for these as well.
https://packages.debian.org/bullseye/fonts-dejavu-core
https://packages.debian.org/bullseye/fontconfig